src/gui/pairPotentialCheckDialog.cpp

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2024 Team Dissolve and contributors
+
+#include "pairPotentialCheckDialog.h"
+
+double PairPotentialCheck::radius = 0.0;
+QMessageBox::StandardButton PairPotentialCheck::reply;
+
+// Retrieve the inscribed sphere radius for all configuration boxes find the smallest
+// Compare smallest radius to pair potential range, return radius if smaller
+
+double PairPotentialCheck::checkPairPotential(std::vector<std::unique_ptr<Configuration>>& currentConfigs, double pairPotentialRange) {
+
+    radius = pairPotentialRange;
+
+    for (const auto& config : currentConfigs) {
+        
+        if (config->box()->inscribedSphereRadius() < radius) {
+
+            radius = config->box()->inscribedSphereRadius();
+        }
+    }
```

Comment:
Because you might not find a configuration which necessitates a change in the pair potential range, this would a good use of `std::optional`.
```suggestion
    std::optional<double> radius;

    for (const auto& config : currentConfigs) {
        
        if (config->box()->inscribedSphereRadius() < radius.value_or(pairPotentialRange)) {

            radius = config->box()->inscribedSphereRadius();
        }
    }
```